src/index.js

```diff
@@ -6,36 +6,83 @@ import Game from './Game'
 import { createStore } from 'redux'
 import './index.css';
 
-Mousetrap.bind('up', () => store.dispatch({type: 'ROTATE'}))
-Mousetrap.bind('left', () => store.dispatch({type: 'LEFT'}))
-Mousetrap.bind('right', () => store.dispatch({type: 'RIGHT'}))
+let pausedBoolean = false
 
-const reducer = ( state = new Game(), action ) => {
-  switch (action.type) {
+const dispatchTick = () => {
+  setTimeout( () => store.dispatch({ type: 'TICK' }), 400 )
+}
+
+const isGameInactive = ( state ) => {
+  return ( pausedBoolean === true ||
+    (state.game1.isGameOver() === true || state.game2.isGameOver() === true ) )
+}
+
+const gameStream = ( state = { 'game1': new Game(), 'game2': new Game() }, action ) => {
+  switch ( action.type ) {
     case 'TICK':
-      const revedState = state.tick()
-      if ( !revedState.isGameOver() || !revedState.isPaused() ) {
-        setTimeout( () => store.dispatch({ type: 'TICK' }), revedState.interval )
+      const revedState = {'game1': state.game1.tick(), 'game2': state.game2.tick() }
+      if ( !revedState.game1.isGameOver() &&
```

Comment:
Consider a more intention-revealing name for the identifier `revedState` - I'm not sure what it is.  Also, consider refactoring this boolean expression into another function with an intention revealing name (`gameCompleted()`? not really sure what this is testing for).